scripts/config/facet-deploy.js

```diff
@@ -77,7 +80,6 @@ async function getFacets(config) {
 
   facetArgs["ConfigHandlerFacet"] = { init: ConfigHandlerFacetInitArgs };
   facetArgs["ExchangeHandlerFacet"] = { init: [], constructorArgs: [protocolConfig.EXCHANGE_ID_2_2_0[network]] };
-  facetArgs["SequentialCommitHandlerFacet"] = { init: [], constructorArgs: [protocolConfig.WETH[network]] };
```

Comment:
why?

Reply:
Temporarily, WETH was set in `configHandler` as part of `protocolAddresses` struct and was not set in `SequentialCommitHandlerFacet` constructor, so this line was removed.

I reverted that change now, so it's again set directly in the facet.